CP05MOAS-GL340/CP05MOAS-GL340_D00010_ingest.csv

```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL340/D00010/merged-from-glider/cp_340*.mrg,CP05MOAS-GL340-00-ENG000000,telemetered,Available,

```

Comment:
Gliders 340, 376 and 379 have cp_340*.mrg, whereas all of the others have cp_*.mrg.  These should be consistent, and it seems the later (without the glider number) is better/safer.

Reply:
Done